docs/extending/extending_page.md

```diff
@@ -455,6 +455,163 @@
     }
     ```
 
+    ### Block name translation
+
+    A practical example of how you can use the `BlockDefinitionEvents` API is translating the block name.
+    You can modify the `name` attribute of the Page block, so that it displays a translation
+    in one of the defined languages.
+
+    The example uses a [Symfony Translator](https://github.com/symfony/symfony/blob/5.1/src/Symfony/Component/Translation/Translator.php) module and its `trans()` method.
+    The method takes three arguments: an identifier of the block name label, an array of parameters,
+    and the domain of the translation.
+
+    Start with adding the requirement to install the language package to the `composer.json` file,
```

Comment:
Maybe use `composer require`, like we do in https://doc.ezplatform.com/en/master/guide/back_office_translations/#installing-new-translations?
Not sure if either is the recommended way.